test/test_sso.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, username=TESTINGNAME, email=DEFAULTEMAIL, sub=TESTINGSUB, gro
     def authorize_access_token(self):
         if self.throws:
             raise self.throws
-        return "Schinken"
+        return {'userinfo': self.parse_id_token(None)}
```

Comment:
With details mentioned in the `sso_auth()` changes of this review, it'll make sense to have a new test case to test for the lack of `userinfo` key of the `authorize_access_token` call.
Orthogonal to the other conditional mock behavior (like `self.thorws`) we should have a `self.has_userinfo` and either return a dict with or without the `userinfo` key set -- plus the matching test case with `has_userinfo = False`.